Description: Fix cross build of the package
 Ignore m4 files included in the upstream source.
Author: Valentin Vidic <vvidic@debian.org>
Bug-Debian: https://bugs.debian.org/931083
Last-Update: 2019-06-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/aclocal.m4
+++ /dev/null
@@ -1,7 +0,0 @@
-m4_include([pkg.m4])
-m4_include([glib-2.0.m4])
-m4_include([runlog.m4])
-m4_include([python.m4])
-m4_include([pythondev.m4])
-m4_include([blkid.m4])
-m4_include([mbvendor.m4])
--- /dev/null
+++ b/acinclude.m4
@@ -0,0 +1,3 @@
+m4_include([pythondev.m4])
+m4_include([blkid.m4])
+m4_include([mbvendor.m4])
--- a/pythondev.m4
+++ b/pythondev.m4
@@ -8,8 +8,8 @@
 [AC_REQUIRE([AM_PATH_PYTHON])
 AC_MSG_CHECKING(for headers required to compile python extensions)
 dnl deduce PYTHON_INCLUDES
-py_prefix=`$PYTHON -c "import sys; print sys.prefix"`
-py_exec_prefix=`$PYTHON -c "import sys; print sys.exec_prefix"`
+py_prefix=`$PYTHON -c "import sys; print(sys.prefix)"`
+py_exec_prefix=`$PYTHON -c "import sys; print(sys.exec_prefix)"`
 PYTHON_INCLUDES="-I${py_prefix}/include/python${PYTHON_VERSION}"
 if test "$py_prefix" != "$py_exec_prefix"; then
   PYTHON_INCLUDES="$PYTHON_INCLUDES -I${py_exec_prefix}/include/python${PYTHON_VERSION}"
